code/game/objects/items/devices/spyglasses.dm

```diff
@@ -12,7 +12,8 @@
 	if(user.client.screen_maps["spypopup_map"]) //alright, the popup this object uses is already IN use, so the window is open. no point in doing any other work here, so we're good.
 		return
 	user.client.setup_popup("spypopup", 3, 3, 2)
-	user.client.register_map_obj(linked_bug.cam_screen)
+	for(var/plane in linked_bug.cam_plane_masters)
+		user.client.register_map_obj(plane)
```

Comment:
```suggestion
	user.client.register_map_obj(linked_bug.cam_screen)
```
Duplicate lines and also breaks the functionality of the spyglass kit, had to fix it upstream
https://github.com/BeeStation/BeeStation-Hornet/pull/8322